variables.tf

```diff
@@ -755,19 +761,25 @@ variable "clusters" {
   description = "A list describing clusters workloads to create"
   type = list(
     object({
-      name               = string           # Name of Cluster
-      vpc_name           = string           # Name of VPC
-      subnet_names       = list(string)     # List of vpc subnets for cluster
-      workers_per_subnet = number           # Worker nodes per subnet.
-      machine_type       = string           # Worker node flavor
-      kube_type          = string           # iks or openshift
-      kube_version       = optional(string) # Can be a version from `ibmcloud ks versions` or `latest`
-      entitlement        = optional(string) # entitlement option for openshift
-      pod_subnet         = optional(string) # Portable subnet for pods
-      service_subnet     = optional(string) # Portable subnet for services
-      resource_group     = string           # Resource Group used for cluster
-      cos_name           = optional(string) # Name of COS instance Required only for OpenShift clusters
-      update_all_workers = optional(bool)   # If true force workers to update
+      name                            = string           # Name of Cluster
+      vpc_name                        = string           # Name of VPC
+      subnet_names                    = list(string)     # List of vpc subnets for cluster
+      workers_per_subnet              = number           # Worker nodes per subnet.
+      machine_type                    = string           # Worker node flavor
+      kube_type                       = string           # iks or openshift
+      kube_version                    = optional(string) # Can be a version from `ibmcloud ks versions` or `latest`
+      logdna_plan                     = optional(string) # Logging plan to provision
```

Comment:
How are the logdna_plan and sysdig_plan used here given that the ocp module does not deploy logdna/sysdig?

Reply:
It was added as part of the earlier change when we deployed logdna & sysdig, but the plans were later drops. Kept the variable thinking it will be introduced in later phases. Hence kept them optional.